test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11562,20 +11585,24 @@ def save_configuration(self, outfile, mode='a'):
         hd = os.path.join(sched_priv, 'holidays')
         self._save_config_file(conf, hd)
 
-        try:
-            with open(outfile, mode) as f:
-                cPickle.dump(sconf, f)
-        except:
-            self.logger.error('error saving configuration ' + outfile)
-            return False
+        self.server.saved_config[MGR_OBJ_SCHED] = conf
+        if outfile is not None:
+            try:
+                with open(outfile, mode) as f:
+                    json.dump(self.server.saved_config, f)
```

Comment:
Since we are dumping `MGR_OBJ_SERVER/SCHED` to the file, would it be better to clean the data saved in `saved_config`? As we are already storing it in file, I don't think we need to keep the same in memory.